CONTRIBUTING.md

```diff
@@ -1,5 +1,8 @@
 If you want to contribute, but aren't sure where to start, you can create a [new discussion](https://github.com/tmm/wagmi/discussions) or reach out [on Twitter](https://twitter.com/awkweb).
 
+### Prerequisites
+This project uses [PnpM](https://pnpm.io/) as a package manager.
```

Comment:
```suggestion
This project uses [pnpm](https://pnpm.io/) as a package manager.
```

They randomize it on their homepage 😛
![CleanShot 2022-01-10 at 18 38 34](https://user-images.githubusercontent.com/6759464/148856129-969bb544-b80b-4fed-8a26-b9a586a8fa9b.gif)